CHANGELOG.next.asciidoc

```diff
@@ -85,6 +85,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Move service config under metrics and simplify metric types. {pull}18691[18691]
 - Fix ECS compliance of user.id field in system/users  metricset {pull}19019[19019]
 - Rename googlecloud stackdriver metricset to metrics. {pull}19718[19718]
+- The Kibana collector applies backoff when errored at getting usage stats {pull}20772[20772]
```

Comment:
This entry is currently in the breaking changes section of the changelog. I think we should move it to the enhancement section instead.